rootless-install.sh

```diff
@@ -304,10 +305,22 @@ do_install() {
 
 	tmp=$(mktemp -d)
 	trap "rm -rf $tmp" EXIT INT TERM
-	# Alternatively could find latest nightly release from https://download.docker.com/linux/static/nightly/ .
-	# Later we can provide different channels.
-	STATIC_RELEASE_URL="https://master.dockerproject.org/linux/x86_64/docker.tgz"
-	STATIC_RELEASE_ROOTLESS_URL="https://master.dockerproject.org/linux/x86_64/docker-rootless-extras.tgz"
+
+	# Download docker binary based on the build the user requested
+	BASE_STATIC_RELEASE_URL="https://download.docker.com/linux/static"
+	MASTER_STATIC_RELEASE_URL="https://master.dockerproject.org/linux/x86_64"
+	if [ -z "$CHANNEL" ] || [ "$CHANNEL" = "test" ]; then
+					STATIC_RELEASE_URL="$BASE_STATIC_RELEASE_URL/test/x86_64/docker-19.03.0-beta4.tgz"
```

Comment:
these need to be automatically detected, not hardcoded

Reply:
detecting the channel or the `docker-19.03.0-beta4.tgz`?

Reply:
try to find the last build url from the list at `download.docker.com` for the specific channel

Reply:
Yeah we should probably also look into having just a latest `docker.tgz` in these as well.

Reply:
We could modify the jobs to upload a `docker.tgz`.  I'll try to find a link to the Jenkin jobs